Site/ASAB Maestro/Files/asab-remote-control/asab-config/General/__schema__.json

```diff
@@ -42,23 +42,23 @@
 		"smtp": {
 			"type": "object",
 			"title": "SMTP Server",
-			"description": "Add configuration for the SMTP server to send emails from the system.",
+			"description": "Configuration for the SMTP server to send emails from the system.",
 			"properties": {
 				"host": {
 					"type": "string",
 					"title": "Host",
-					"description": "Where is your SMTP server.",
-					"examples": "https://smtp-provider.com/"
+					"description": "Specify the domain name or IP address of your SMTP server.",
+					"examples": "smtp.example.com"
 				},
 				"user": {
 					"type": "string",
 					"title": "User",
-					"description": "Set credentials for the SMTP server. Type 'apikey' and fill in apikey into password option if needed."
```

Comment:
this (using "apikey" and user) is specific for sendgrid and specified in sendgrid documentation, not implemented anywhere in our services